roles/mysql_hardening/CHANGELOG.md

```diff
@@ -1,4 +1,10 @@
 # Changelog
+## []() ()
+
+**Implemented enhancements:**
```

Comment:
The changelog gets automatically generated from the PRs, there's no need to create it manually.